### PR TITLE
Update troubleshooting section to collect networking logs.

### PR DESCRIPTION
**Pull request by krishnacx**

### CODE REVIEW

File: defender-endpoint/mde-plugin-wsl.md [hunk 1]
```diff
@@ -293,7 +296,30 @@ DeviceProcessEvents
...
+      
+      4.	Open an elevated(admin) command prompt and run:  `wsl --debug-shell`
+
+      5.	In debug shell, run: `mdatp connectivity test`
+
+      6.	Allow the connectivity test to be completed
+
+      7.	Stop the .ps1 ran in step #2
+
+      8.	Share the generated .zip file along with support bundle.
```
Comment (hiwarkarvinay) [inline defender-endpoint/mde-plugin-wsl.md]: Mention cmd to collect support bundle in a separate step or link to existing doc on how to do it.

**Comment by krishnacx** [comment]
@pahuijbr can we merge the PR ?

**Comment by krishnacx** [comment]
@denise-msft Kindly merge the PR

**Comment by pahuijbr** [comment]
@Denise ***@***.***> not sure how this got assigned to me, but please go ahead and review and merge
________________________________

**Review by denisebmsft — APPROVED** [review]
Thank you @krishnacx

**Pull request merged**